client/src/selectors/schedules.js

```diff
@@ -23,6 +50,19 @@ export const selectSchedules = (schedules, { startTime, endTime }) => {
   return filteredItems.sort((a, b) => (a.startTime > b.startTime ? 1 : -1));
 };
 
+export const selectColor = (status) => {
+  switch (status) {
```

Comment:
faster, easier test coverage:
```
const colors = {
  'Available': Colors.bgBtnAvailable,
  'Unavailable': Colors.bgBtnUnavailable,
  'Urgent': Colors.bgBtnUrgent,
};
const color = colors[status];
if (typeof colors === 'undefined') {
  return Color.bgWhite;
}
return color;
```
now you don't need to get coverage on every branch in the switch.

Reply:
Updated, 

export const selectColor = (status) => {
  const colors = {
    Available: Colors.bgBtnAvailable,
    Unavailable: Colors.bgBtnUnavailable,
    Urgent: Colors.bgBtnUrgent,
  };
  const color = colors[status];
  if (typeof colors === 'undefined') {
    return Colors.bgWhite;
  }
  return color;
};